lti_consumer/tests/unit/test_outcomes.py

```diff
@@ -370,6 +370,20 @@ def test_grade_past_due(self):
         self.assertIn('failure', response)
         self.assertIn('Grade is past due', response)
 
+    @patch('lti_consumer.outcomes.parse_grade_xml_body')
+    def test_lti_error_not_raises_type_error(self, mock_parse):
```

Comment:
Does this test fails without the fix with error `a bytes-like object is required`?

Reply:
Yes! it would be failed without the fix that is going to be added!